ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml

```diff
@@ -217,6 +217,23 @@ tests:
       MCE_VERSION: "2.4"
       ODF_OPERATOR_SUB_CHANNEL: stable-4.14
     workflow: hypershift-mce-kubevirt-baremetalds-conformance
+- as: e2e-kubevirt-metal-mce-disconnected
+  cron: 0 8 * * *
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      HYPERSHIFT_NODE_COUNT: "2"
+      MCE_VERSION: "2.4"
+      ODF_OPERATOR_SUB_CHANNEL: stable-4.14
+    workflow: hypershift-mce-kubevirt-baremetalds-disconnected-conformance
+- as: e2e-kubevirt-metal-disconnected
+  cron: 0 8 * * *
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      HYPERSHIFT_NODE_COUNT: "2"
+      ODF_OPERATOR_SUB_CHANNEL: stable-4.14
+    workflow: hypershift-kubevirt-baremetalds-disconnected-conformance
```

Comment:
I don't think we need this job. the only time we're not testing MCE in the periodics is when we're working with the latest dev branch

Reply:
Correct, it was only to verify the non MCE flow on 4.14.
Let's drop it.